README.md

```diff
@@ -8,7 +8,136 @@ pod 'IntrepidSwiftWisdom'
 use_frameworks!
 ```
 
-####Contributions
+### Components
+
+* [Qu](#qu)
+* [After](#after)
+* [Nib Initable](#nib-initable)
+* [Color Descriptor](#color-descriptor)
+* [Cell Configuring](#cell-configuring)
+* [User Defaults](#user-defaults)
+
+#### Qu
+
+A basic wrapper for dispatch operations in Swift.  Syntax example:
+
+```Swift
+    Qu.Background {
+        // Sleep for long operation
+        sleep(4)
+        log("1")
+    } .Also {
+        sleep(4)
+        log("2")
+    } .Also {
+        sleep(1)
+        log("3")
+    } .Also {
+        sleep(1)
+        log("4")
+    } .Also {
+        sleep(1)
+        log("5")
+    } .ThenAfter(.Previous(3)) {
+        log("6: After 5, 4, & 3")
+    } .Then {
+        sleep(1)
+        log("7: After 6")
+    } .FinallyOn(.Main) {
+        sleep(1)
+        log("Finished: After All")
+    }
+```
+
+#### After
+
+A simple way to perform or repeat future operations:
+
+```Swift
+After(2.5) {
+  print("Two and a half seconds later")
+}
+
+RepeatAtInterval(1.0, numberOfTimes: 5) {
+  print("Once a second, 5 times")
+}
+```
+
+#### Nib Initable
+
+Load views from nibs:
+
+```Swift
+let myCustomView = MyCustomView.fromNib()
+```
+
+#### Color Descriptor
+
+Setup Easy Color Schemes
+
+```Swift
+enum ColorPalette : ColorDescriptor {
+    case White = "254,216,130,255"
+    case DarkGreen = "51,58,24,255"
+    case DarkGray = "64,48,56,255"
+
+    var color: UIColor {
+        return rawValue.color
+    }
+}
+```
+
+And use:
+
+```Swift
+someView.backgroundColor = ColorPalette.White.color
```

Comment:
I feel like its worth noting somewhere that zeplin does generate swift color palletes, though this might not be the right place for it